packages/wallets/torus/src/adapter.ts

```diff
@@ -114,34 +103,31 @@ export class TorusWalletAdapter extends BaseSignerWalletAdapter {
     }
 
     async disconnect(): Promise<void> {
-        const openLogin = this._openLogin;
-        if (openLogin) {
-            this._openLogin = null;
-            this._keypair = null;
-
+        // torus.logout
+        const wallet = this._torus;
+        if (wallet) {
+            window.torus = null
```

Comment:
I'm a little bit sketched out by setting and unsetting window globals like this -- is this something that the Torus Embed code expects?